docs/basic-features/image-optimization.md

```diff
@@ -121,7 +121,7 @@ The following Image Optimization cloud providers are included:
 - [Imgix](https://www.imgix.com): `loader: 'imgix'`
 - [Cloudinary](https://cloudinary.com): `loader: 'cloudinary'`
 - [Akamai](https://www.akamai.com): `loader: 'akamai'`
-- dangerously-unoptimized: uses image directly from `src` prop without optimization (only needed with `next export` or when the built-in `_next/image` optimizer can not be leveraged) `loader: 'dangerously-unoptimized'`
+- Custom: `loader: 'custom'` use a custom cloud provider by implementing the [`loader`](/docs/api-reference/next/image.md#loader) prop on the `next/image` component
```

Comment:
```suggestion
- Custom: use a custom cloud provider by implementing the [`loader`](/docs/api-reference/next/image.md#loader) prop on the `next/image` component  `loader: 'custom'`
```
Might be good to keep this at the end to match the above lines

Reply:
I think it looks fine in the front so I'll keep it as-is 🙂 

![image](https://user-images.githubusercontent.com/229881/124975668-3e01f380-dffc-11eb-9ead-4d176456a00d.png)